Tests/LibWeb/Text/expected/all-window-properties.txt

```diff
@@ -338,6 +338,9 @@ SVGDefsElement
 SVGDescElement
 SVGElement
 SVGEllipseElement
+SVGFEBlendElement
+SVGFEFloodElement
+SVGFEGaussianBlurElement
```

Comment:
These test updates should go in the commits that are causing them to change, i.e. make the commits atomic.